src/Core/Injector/Injector.php

```diff
@@ -527,7 +527,13 @@ public function convertServiceProperty($value)
 
         // Evaluate constants surrounded by back ticks
         if (preg_match('/^`(?<name>[^`]+)`$/', $value, $matches)) {
-            $value = defined($matches['name']) ? constant($matches['name']) : null;
+            if (defined($matches['name'])) {
+                $value = constant($matches['name']);
+            } elseif (getenv($matches['name'])) {
```

Comment:
I think this should be a `!== false` check. getenv will return false if the var isn't set; if it's set to a blank it will return an empty string.

Tested with this:

```php
<?php
echo getenv("TEST") === false ? "F" : (getenv("TEST") === "" ? "Emptystring" : "Neither F nor emptystring");
```
Result:
```
> env TEST= php test.php 
Emptystring
> env TEST=false php test.php 
Neither F nor emptystring
> php test.php 
F
> env TEST=hello php test.php 
Neither F nor emptystring